src/riak_kv_pb_timeseries.erl

```diff
@@ -818,20 +824,18 @@ make_tscolumndescription_list(ColumnNames, ColumnTypes) ->
     [#tscolumndescription{name = Name, type = riak_pb_ts_codec:encode_field_type(Type)}
      || {Name, Type} <- lists:zip(ColumnNames, ColumnTypes)].
 
-
-
-decoder_parse_error_resp({LineNo, riak_ql_parser, Msg}) when is_integer(LineNo) ->
-    flat_format("~ts", [Msg]);
-decoder_parse_error_resp({Token, riak_ql_parser, _}) ->
-    flat_format("Unexpected token '~p'", [Token]);
-decoder_parse_error_resp(Error) ->
+make_decoder_error_response({LineNo, riak_ql_parser, Msg}) when is_integer(LineNo) ->
+    make_rpberrresp(?E_PARSE_ERROR, flat_format("~ts", [Msg]));
+make_decoder_error_response({Token, riak_ql_parser, _}) when is_binary(Token) ->
+    make_rpberrresp(?E_PARSE_ERROR, flat_format("Unexpected token '~s'", [Token]));
```

Comment:
Do we know that `Token` will always be a string? Just preparing for the degenerate case in which something unexpected actually blows up the error statement.